_posts/platform/internals/2000-01-01-billing.md

```diff
@@ -4,10 +4,20 @@ modified_at: 2019-03-22 00:00:00
 tags: billing
 ---
 
+## Fill the Billing Profile
+
+In order to provision larger than Free-tiers resources, you have to fill your billing profile.
+
+For that, go to https://billing.scalingo.com and fill the form.
+
+![Gif: how to fill the Billing Profile](https://i.imgur.com/QippVrK.gif)
```

Comment:
```suggestion
![how to fill the Billing Profile](https://i.imgur.com/QippVrK.gif)
```

Not sure it's relevant to add the format in the description.

Reply:
it's for blind people

Reply:
If you use gifs, please upload them on our CDN (ask for access) and not on a third party site.

Reply:
Ok. Can I have access ? :D